### ISSUE TITLE
Rule `naming-convention` - FieldDefinition camelCase not honored for input fields

### ISSUE THREAD

**Issue by ahernandez2-rbi** [B2o5T/graphql-eslint#1130]
### Issue workflow progress

_Progress of the issue based on the [Contributor Workflow](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#a-typical-contributor-workflow)_

- [ ] 1. The issue provides a reproduction available on GitHub, Stackblitz or CodeSandbox
> Please make sure the graphql-eslint version under `package.json` matches yours.
- [ ] 2. A failing test has been provided
- [ ] 3. A local solution has been provided
- [ ] 4. A pull request is pending review

---

**Describe the bug**

Given below configuration and invalid input, the naming-convention rule should flag the fields of the input Foo.

```graphql
# eslint @graphql-eslint/naming-convention: ['error', { types: 'PascalCase', FieldDefinition: 'camelCase' }]

input Foo {
  VALUE: String!
  Value: String!
  va_lue: String!
}
```

Package version: `"@graphql-eslint/eslint-plugin": "3.10.7"`

**To Reproduce**
See above

**Expected behavior**
All fields of `input Foo` to be flagged as invalid because fields are configured with camelCase.

**Environment:**

- OS: Mac
- `@graphql-eslint/eslint-plugin`: 3.10.7
- Node.js: v14.17.3

**Comment by ahernandez2-rbi** [comment]
This is actually configured under InputValueDefinition
